Bug#520919: icon-naming-utils: wrong icons for NetworkManager

Fix symbolic kinks for the NetworkManager status icons.

Signed-off-by: Josselin Mouette <joss@debian.org>

Index: icon-naming-utils-0.8.90/legacy-icon-mapping.xml
===================================================================
--- icon-naming-utils-0.8.90.orig/legacy-icon-mapping.xml	2009-03-23 16:37:28.499379079 +0100
+++ icon-naming-utils-0.8.90/legacy-icon-mapping.xml	2009-03-23 16:39:12.312936039 +0100
@@ -741,9 +741,12 @@
 	</icon>
 	<icon name="network-wired">
 	    <link>gnome-dev-ethernet</link>
+	    <link>nm-device-wired</link>
 	</icon>
 	<icon name="network-wireless">
 	    <link>gnome-dev-wavelan</link>
+	    <link>nm-adhoc</link>
+	    <link>nm-device-wireless</link>
 	</icon>
 	<icon name="phone">
 	    <link>stock_cell-phone</link>
@@ -1171,9 +1174,6 @@
 	</icon>
 	<icon name="network-idle">
 	    <link>gnome-netstatus-idle</link>
-	    <link>nm-adhoc</link>
-	    <link>nm-device-wired</link>
-	    <link>nm-device-wireless</link>
 	    <link>connect_established</link>
 	</icon>
 	<icon name="network-offline">
